### PR TITLE
Update rowrankdensefunction.md

### PR DESCRIPTION
**Pull request by wilkovandevelde**
Added optional parameter: restart

### CODE REVIEW

**Comment by wilkovandevelde** [comment]
@microsoft-github-policy-service agree

File: data-explorer/kusto/query/rowrankdensefunction.md [hunk 1]
```diff
@@ -54,3 +55,33 @@ SEA      | LH       | 3           | 2
 SEA      | UA       | 3           | 2
 SEA      | EL       | 3           | 2
 SEA      | LY       | 100         | 3
+
```
Comment (v-dkorn) [inline data-explorer/kusto/query/rowrankdensefunction.md:58]:
```suggestion

> [!div class="nextstepaction"]
> <a href="https://dataexplorer.azure.com/clusters/help/databases/Samples?query=H4sIAAAAAAAAA2WQTQuCQBCG7wv+h8mTggetm+DBSOhgl6RDRIi6g0iyyuz2Bf341khtiYGXmYd3hpnhhdJRtghO3FDfkQqlokbUHui6bQRO9Qb7gtSVUIZtJ2rXYieLAdhZEtse2OlW68oz0FFr4PsGPMR/vvWgSwMl6eyKd9mEAgN9GgOLnS32AqmXh/IJ3zuAo6x+t4ZCVoMPHwoFh30hLhF195x0knMUEp3Z7EFPeBt/4sIiGue6b3gIxbszAQAA" target="_blank">Run the query</a>
```

**Review by v-dkorn — APPROVED** [review]

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**